lib/ansible/module_utils/network/ios/config/l3_interfaces/l3_interfaces.py

```diff
@@ -204,6 +204,34 @@ def _state_deleted(self, want, have):
 
         return commands
 
+    def check_diff_again(self, want, have):
+        """
+        Check the IPV4 difference again when secondary IP is configured,
```

Comment:
Could we rephrase this? I think it's a little bit confusing. IIUC when there's a difference in order there'd be a mismatch on the set? Too much difference on the comment ;)